### PR TITLE
Project: Energy Efficiency and Load Balancing

### PR DESCRIPTION
**Pull request by isabelheisters**
Signed-off-by: Isabel Heisters i.heisters@fz-juelich.de

### CODE REVIEW

**Comment by pancetta** [comment]
There are conflicts in your PR. It seems that you'd need to update your local branch first (right, @torbjoernk)?

**Comment by torbjoernk** [comment]
Jep, probably a conflict in `jlesc.bib`.

**Comment by isabelheisters** [comment]
What should I do to repair this problem? I changed the comments in the jlesc.bib, so this shouldn't be a problem anymore.

File: _bibliography/external/eandb_project.bib [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+  url       = {http://dx.doi.org/10.1049/iet-cdt.2014.0074},
+  volume    = {9},
+  year      = {2015},
+}
+
+
+
+@inproceedings{CastroEtAl2014,
+  author    = {Marcio Bastos Castro and Fabrice Dupros and Emilio Francesquini and
+   Jean-Francois Mehaut andPhilippe Olivier Alexandre Navaux},
```
Comment (pancetta) [inline _bibliography/external/eandb_project.bib]: Missing blank after last `and`.

File: _projects/eandb_project.md [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+Our strategy employs an Energy Daemon Tool to gather power information and a load balancing module
+that benefits from the load balancing framework available in the CHARM++ runtime system. 
+We propose two variants of our energy-aware load balancer (ENERGYLB) to save energy on imbalanced
+workloads without considerably impacting the overall system performance. 
+The first one, called Fine- Grained EnergyLB (FG-ENERGYLB), is suitable for plat- forms composed of few
+tens of cores that allow per-core DVFS. 
+The second one, called Coarse-Grained EnergyLB (CGENERGLB) is suitable for current HPC platforms composed of several 
+multi-core processors that feature per-chip DVFS.  
+This work was presented at the HiPC conference {% cite PadoinEtAl2014 --filse external/eandb_project.bib %}.
+
```
Comment (pancetta) [inline _projects/eandb_project.md:96]: `--filse` -> `--file`

File: _projects/eandb_project.md [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+different interfaces to collect data, making it difficult to correlate performance with energy consumption.
+To overcome this issue, we developed a platform-independent tool that collects power
+and energy data from homogeneous and heterogeneous systems. 
+Using this tool, it provides a detailed comparison between a low-power processor (ARM big.LITTLE) and a high performance
+processor (Intel Sandy Bridge-EP) using all applications from the NAS parallel benchmarks and
+a real-world soil irrigation simulator. 
+The results show that the average power demand of Intel Sandy Bridge-EP is within 12:6X to 152:4X higher than ARM big.
+LITTLE, whereas its average energy consumption is within 1:6X to 7:1x superior. 
+Overall, ARM big.LITTLE presented a better performance/energy trade-off when it takes less than 9:2X the execution 
+time of Intel Sandy Bridge-EP to solve the same problem.
```
Comment (pancetta) [inline _projects/eandb_project.md:68]: These `:` are simple `.`.

File: _projects/eandb_project.md [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+The decomposition of the domain into a grid of subdomains, which are distributed among tasks, creates 
+load differences between the tasks that simulate the borders and those responsible for the central subdomains. 
+Second, the propagation of waves in the simulated area changes the workload on the subdomains on different time-steps. 
+Therefore causing dynamic load imbalance. 
+In order to evaluate the use of dynamic load balancing, we ported a seismic wave simulator to Adaptive MPI, 
+to benefit from its load balancing framework. Our experimental results
+show that dynamic load balancers can adapt to load variations during the application’s execution
+and improve performance by 36%.
+This work was presented in the PDP 2014 conference {% cite KellerTesserEtAl2014 --file jlesc.bib %}. 
+An extended version will be publishedin the International Journal of High Performance Computing and Applications.
```
Comment (pancetta) [inline _projects/eandb_project.md]: In the original report, they cite another reference here..

**Comment by torbjoernk** [comment]
@isabelheisters, switch to your local source branch, pull all upstream changes in, the switch back to your feature branch and merge your local source branch into your feature branch:

```
git checkout source
git fetch --all --prune
git pull
git checkout FEATURE_BRANCH
git merge --no-ff --no-commit --log source
```

Now resolve all conflicts manually by editing the conflicting files. Finally, add your changges and commit and push the merge:

```
git add -A
git commit --signoff
git push
```

File: _projects/eandb_project.md (hunk 4 above)
Comment (isabelheisters) [inline _projects/eandb_project.md]: Did you mean the reference in line 50? In the report I have is a reference to: Improving the Performance of Seismic Wave Simulations with Dynamic Load Balancing. Is this the wrong?

Comment (pancetta) [inline _projects/eandb_project.md]: I'm talking about their ref. [6], "Dynamic load balancing for seismic wave propagation models".. this one is not showing up here.

Comment (isabelheisters) [inline _projects/eandb_project.md]: The only information I have found was this one: https://team.inria.fr/exase/international-journal-of-high-performance-computing-applications/. I thought it was not published at the moment.

Comment (pancetta) [inline _projects/eandb_project.md]: That's ok, you can also cite paper with "submitted" or "accepted" (like this one).

**Comment by isabelheisters** [comment]
So I tried to fix the problems.

**Pull request merged**